ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -130,10 +129,7 @@ module.exports = Marionette.LayoutView.extend({
                     break;
                 case 'MULTIPOLYGON':
                 case 'POLYGON':
-                    this.model.set({
-                        mode: 'poly',
-                        polygon: filter.polygon,
-                    });
+                    this.handlePolygonDeserialization(filter);
                     wreqr.vent.trigger('search:polydisplay', this.model);
```

Comment:
Should this be removed since it's called in `handlePolygonDeserialization`?

Reply:
Yes! Thanks @jrnorth